plugin/src/main/kotlin/com/ritense/plugin/service/PluginService.kt

```diff
@@ -95,6 +95,13 @@ class PluginService(
         return pluginConfigurationSearchRepository.search(pluginConfigurationSearchParameters)
     }
 
+    fun getPluginConfigurationsForClass(
```

Comment:
We already have this method:
`fun <T> findPluginConfigurations(clazz: Class<T>, filter: (JsonNode) -> Boolean = { true }): List<PluginConfiguration>`. Why not use that one?

Reply:
Because it was created a month after this was initially written. I'll remove mine.